docassemble/EFSPIntegration/efm_client.py

```diff
@@ -509,3 +511,34 @@ def get_disclaimers(self, court_id:str):
     send = lambda: self.proxy_client.get(self.base_url + f'codes/courts/{court_id}/disclaimer_requirements')
     return self._call_proxy(send)
   
+
+def serialize_person(person:Union[Person,Individual])->Dict:
+  """
+  Converts a Docassemble Person or Individual into a dictionary suitable for
+  json.dumps and in format expected by the EFSPProxy
```

Comment:
```suggestion
  json.dumps and in format expected by Tyler-specific endpoints on the EFSPProxy
```
There's confusingly multiple types of people that we can send